packages/core/admin/admin/src/components/FormInputs/Date.tsx

```diff
@@ -1,6 +1,5 @@
-import { forwardRef, memo } from 'react';
+import { forwardRef, memo, useState } from 'react';
```

Comment:
Could you update this import to `import * as React`, then use React.useState etc? it's not enforced by ESLint yet but it's the style we try to apply everywhere for consistency